spec/models/linked_data/linked_ephemera_folder_spec.rb

```diff
@@ -96,14 +96,22 @@
 
   describe '#subject' do
     context 'with Valkyrie::IDs for values' do
-      let(:ephemera_folder) { FactoryGirl.create_for_repository(:ephemera_folder, subject: [ephemera_term.id]) }
+      let(:ephemera_folder) { FactoryGirl.create_for_repository(:ephemera_folder, subject: [ephemera_child_term.id]) }
+      let(:ephemera_vocabulary) { FactoryGirl.create_for_repository(:ephemera_vocabulary, uri: 'https://example.com/ns/testVocabulary') }
+      let(:ephemera_child_term) { FactoryGirl.create_for_repository(:ephemera_term, label: 'test child term', member_of_vocabulary_id: ephemera_vocabulary.id) }
       it 'exposes the values as JSON-LD Objects' do
         expect(linked_ephemera_folder.subject).not_to be_empty
         expect(linked_ephemera_folder.subject.first).to eq(
-          "@id" => "http://www.example.com/concern/ephemera_terms/#{ephemera_term.id}",
+          "@id" => "http://www.example.com/concern/ephemera_terms/#{ephemera_child_term.id}",
           "@type" => "skos:Concept",
-          "pref_label" => ephemera_term.label,
-          "exact_match" => { "@id" => ephemera_term.uri.first }
+          "pref_label" => ephemera_child_term.label,
+          "exact_match" => { "@id" => ephemera_child_term.uri.first },
+          "in_scheme" => {
+            "@id" => "http://www.example.com/concern/ephemera_vocabularies/#{ephemera_vocabulary.id}",
+            "@type" => "skos:Concept",
```

Comment:
Shouldn't this be a `skos:ConceptScheme`?

Reply:
Probably, fixing.